scenario_test_tools/scripts/example_move_base.py

```diff
@@ -0,0 +1,29 @@
+#! /usr/bin/env python
+
+import rospy
+
+from scenario_test_tools.scriptable_move_base import ScriptableMoveBase
+
+from move_base_msgs.msg import MoveBaseAction, MoveBaseResult
+
+
+if __name__ == "__main__":
+    rospy.init_node('move_base_fake')
+
+    succeeded = MoveBaseResult()
+
+    result_delay = rospy.get_param("result_delay", 5)  # By default, wait 5 secs for the robot to arrive
+
+    move_base = ScriptableMoveBase('/move_base', MoveBaseAction, result_delay)
```

Comment:
I think @HannesBachter needs the action name to be configurable

Reply:
Indeed, but the goal here is to make a canonical example. But renaming is of course possible through remapping:
```bash
rosrun scenario_test_tools example_move_base.py /move_base:=/move_base_linear
```

Reply:
With the now private namespace, the remapping should be this:
```bash
rosrun scenario_test_tools example_move_base.py ~move_base:=/move_base_linear
```

Reply:
If you ask my five cents:
remapping is less intuitive then specifying the action name explicitly via a parameter...especially if you want this to be a canonical example for the open source community...
especially since now the action name and the parameter are retrieved from different namespaces...
I'd suggest to query both `result_delay` and `action_name` from private namespace:
```
result_delay = rospy.get_param("result_delay", 5)
action_name = rospy.get_param("action_name", "/move_base")
```